common/cb_types.txt

```diff
@@ -1431,7 +1431,7 @@ liberate_country = {
 
 	badboy_factor = 15
 	prestige_factor = 1
-	peace_cost_factor = 1
+	peace_cost_factor = 0.9
```

Comment:
What's with this change?

Reply:
It was supposed to bring the warscore cost down. Didn't work, if the liberatable country is big enough it's 100 and the AI always white peaces. Is there a way to cap the max warscore for a CB at les than 100?

Reply:
There isn't a good solution. Basically, the `po_transfer_provinces = yes` will need to get replaced with a wargoal that's easier to enforce, like `po_remove_prestige = yes` , and the on_po_accepted will need to handle the liberating of the targeted country. 

We can then set the `peace_cost_factor` to be whatever value we want. However, it will be that value regardless of the size of the tag getting released -- a tag with a random 10k pop province would have the same warscore cost as a tag with 50 provinces and 1 mil pops

Reply:
Is there a way to change how much warscore AI is willing to accept? It seems that you accidentally lowered it because this bug doesn't exist in vanilla Vic2.

Reply:
Afaik, I haven't touched any of that. The AI takes into account the PEACE_... items in the ai section of the defaines when determining when it peaces out of wars

Reply:
Thanks. This bug has existed for years so maybe you just don't remember changing it.

Reply:
I restored the vanilla values in a recent commit